frontend/src/index.js

```diff
@@ -0,0 +1,55 @@
+let loggedInAccount
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
should there be a function call happening inside here once the page loads or can we remove this?

Reply:
we don't really want to do anything when the page loads, instead we are waiting for user input. I can remove this function for now since we don't really fire off any requests immediately.
Potentially we might want to start fetching our accounts and wordbanks here though.